### DISCUSSION TITLE
Can't get used storage from Dropbox

### DISCUSSION THREAD

**Discussion by rezasys2** [spatie/laravel-backup#1433] [Q&A]
Hi
I want to use the 'delete_oldest_backups_when_using_more_megabytes_than' option to automatically keep my Dropbox usage under 2 gigs, but it seems that the package can't get the used storage from Dropbox to do it. It always says "Used storage after cleanup: 0 B." for Dropbox disk. 
The other possible option for me is to keep backups for 4 hours, but I don't know how to implement it.
How can I tackle this issue?

**Comment by erikn69** [comment]
That is a question for the Dropbox package you are using. there are several, no one can use mental powers to guess which one you use

-----

For better question on dropbox package:
https://github.com/spatie/laravel-backup/blob/93874c794a52c3298d9301d433641aedaca4ea7c/src/BackupDestination/Backup.php#L66
https://laravel.com/docs/8.x/filesystem#file-metadata
